lib/MusicBrainz/Server/EditSearch/Query.pm

```diff
@@ -44,9 +43,12 @@ my %field_map = (
     editor_flag => 'MusicBrainz::Server::EditSearch::Predicate::EditorFlag',
     applied_edits => 'MusicBrainz::Server::EditSearch::Predicate::AppliedEdits',
 
-    map {
-        $_ => 'MusicBrainz::Server::EditSearch::Predicate::' . camelize($_)
-    } qw( area artist instrument label place recording release release_group series work )
+    entities_with(['mbid', 'relatable'],
```

Comment:
Note that this adds not only `event`, but `url`, too. You can only use it if you write a suitable query string by hand (I tested it, it works), but it is a first step towards MBS-7610.